src/docs/development/packages-and-plugins/using-packages.md

```diff
@@ -1,28 +1,31 @@
 ---
 title: Using packages
-description: Learn how to use packages in your Flutter app.
+description: How to use packages in your Flutter app.
 ---
 
 Flutter supports using shared packages contributed by other developers
 to the Flutter and Dart ecosystems. This allows you to quickly build
 your app without having to develop everything from scratch.
 
 Existing packages enable many use cases, for example, making network requests
-([`http`](/docs/cookbook/networking/fetch-data)), custom navigation/route handling
-([`fluro`]({{site.pub-pkg}}/fluro)), integration with device
-APIs (like [`url_launcher`]({{site.pub-pkg}}/url_launcher) &
+([`http`](/docs/cookbook/networking/fetch-data)),
+custom navigation/route handling ([`fluro`]({{site.pub-pkg}}/fluro)),
+integration with device APIs
+([`url_launcher`]({{site.pub-pkg}}/url_launcher) and
 [`battery`]({{site.pub-pkg}}/battery)),
-and using third-party platform SDKs (like
-[Firebase]({{site.github}}/flutter/plugins/blob/master/FlutterFire.md)).
+and using third-party platform SDKs
+([Firebase]({{site.github}}/flutter/plugins/blob/master/FlutterFire.md)).
```

Comment:
Firebase -> FlutterFire?

If not, then maybe:
SDKs like Firebase ([FlutterFire](...)).